journal/src/main/scala/com/evolutiongaming/kafka/journal/eventual/ReplicatedKeyJournal.scala

```diff
@@ -18,19 +18,19 @@ trait ReplicatedKeyJournal[F[_]] {
     timestamp: Instant,
     expireAfter: Option[ExpireAfter],
     events: Nel[EventRecord]
-  ): F[Unit]
+  ): F[Boolean]
```

Comment:
can we replace `Boolean` everywhere with type alias `Updated` or `Changed` ?

Reply:
good point